samples/display/lvgl/boards/reel_board.conf

```diff
@@ -1,9 +0,0 @@
-CONFIG_SPI=y
```

Comment:
How is SPI enabled now?

Reply:
It was already enabled at board level:
https://github.com/zephyrproject-rtos/zephyr/blob/d5072d13786edc4bc34a169e743d5aeeee34a819/boards/arm/reel_board/reel_board_defconfig#L14

Same for the display:
https://github.com/zephyrproject-rtos/zephyr/blob/d5072d13786edc4bc34a169e743d5aeeee34a819/boards/arm/reel_board/Kconfig.defconfig#L29-L31